_episodes_rmd/60-next-steps.Rmd

```diff
@@ -23,26 +23,26 @@ library("tidyverse")
 ```
 
 Data in bioinformatics is often complex.
-To deal with this, developers define specialised
+To deal with this, developers define specialized
```

Comment:
This isn't a typo. We use British spelling in this lesson.

Reply:
done 🔨